zynautoconnect/zynthian_autoconnect.py

```diff
@@ -510,19 +509,33 @@ def audio_autoconnect(force=False):
 				continue
 
 			#Get Root Layer Input ports ...
-			rl_in=jclient.get_ports(rl.get_jackname(), is_input=True, is_audio=True)
-
-			#Connect System Capture to Root Layer ports
-			if len(rl_in)>0:
-				if len(rl_in)==1:
-					rl_in.append(rl_in[0])
-
-				try:
-					jclient.connect(system_capture[0],rl_in[0])
-					jclient.connect(system_capture[1],rl_in[1])
-				except:
-					pass
+			rl_in=get_input_ports(rl)
 
+			try:
```

Comment:
the `try:` should also go here (was for debugging only).

However, I found that if `audio_autoconnect()` throws an exception, two problems occur:
- The exception doesn't make its way into the systemd journal logs (perhaps it's eaten or delayed?)
- The mutex is never released so the UI freezes

That's for a different PR of course; any ideas on how to approach this?